### ISSUE TITLE
[Feature Request] MtoN relation in relation map.

### ISSUE THREAD

**Issue by light42** [zadam/trilium#507]
Some relations I created in relation map is not always 1to1. So it's very annoying when I must create identical relations over and over again.

**Comment by zadam** [comment]
Can you clarify what you mean? How would you define let's say 1:N relation? You need to specify which nodes are part of the relation anyway, no?

**Comment by light42** [comment]
Assume that we have this relations:

        |--------isParentOf--> x
        |
Father --isParentOf--> y
        |
        |--------isParentOf--> z

I want that we could merge those three relations so the end result would be like this:

                     |-------> x
                     |
Father --isParentOf ------> y
                     |
                     |-------> z

**Comment by light42** [comment]
Suppose that we have this relations:
![1](https://user-images.githubusercontent.com/6873314/57070060-8462b600-6d00-11e9-9094-2d11ad3f39bf.png)

I want to merge these relations such that the end result would be like this:
![2](https://user-images.githubusercontent.com/6873314/57070094-9a707680-6d00-11e9-940b-439458d71556.png)

**Comment by zadam** [comment]
So if I understand this correctly, you'd prefer if relations from one (to one) note with the same name would be grouped in this way. I agree it does look better/more organized, especially in certain types of diagrams. I'm not sure if the library I use for relation maps (jsPlumb) supports this though.

**Comment by light42** [comment]
@zadam yes, that's exactly what I meant. It's a little bit frustrating when the map gets cluttered by redundant relations.
